cluster/stone-prd-rh01/tenants/ros-tenant/integration_test_scenario.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  name: ros-enterprise-contract
+  namespace: ros-tenant
+spec:
+  params: 
+    - name: POLICY_CONFIGURATION
+      value: rhtap-releng-tenant/app-interface-standard
+  application: ros-backend
```

Comment:
As per the documentation we only need one file `integration_test_scenario.yaml` and it has one application defined! I like to know I have two apps `ros-backend` and `ros-ocp-backend` so in that case what is the practice? As I see this file in other tenants however its just mentioning one service. At the same time I like to know if we still need this as from UI I have configured the parameter POLICY_CONFIGURATION and if that is enough?

Reply:
One integration test per application. You could do it here or directly in the UI, that's really your decision. The important thing to notice is that if you do it here, it will supersede what you did in the UI